supersuit/utils/agent_indicator.py

```diff
@@ -8,17 +8,15 @@ def change_obs_space(space, num_indicators):
     if isinstance(space, Box):
         ndims = len(space.shape)
         if ndims == 1:
-            pad_space = np.ones((num_indicators,), dtype=space.dtype)
+            pad_space = np.max(space.high) * np.ones((num_indicators,), dtype=space.dtype)
```

Comment:
It unify the high and low range of the indicator channel with the original observation channels.
Because sometimes the values original observation channel are in range of [0,255] (Before normalized)

Reply:
Is it possible to instead do `np.min(space.high)`? This is in the very rare but potentially possible case of `space.high = [1, 255, 255]`, `pad_space` will then not be contained within the observation space.

You could implement it to check that all `space.high` are the same values `np.all(space.high[0] == space.high)`, and if they aren't then raise a warning. Doing this within the wrapper function would incur overhead cost though, so I think it's better to put this assert in the wrapper init, and then just do `np.min(space.high)` everywhere else.